rfcs/text/0018/threat.yml

```
@@ -65,6 +65,15 @@
 
     example: "Tool"
 
+- name: group.alias
+  level: extended
+  type: keyword
+  short: Alias of the group.
+  description: >
+    The alias(es) of the group for a set of related intrusion activity that are tracked by a common name in the security community. While not required, you can use a MITRE ATT&CK® group alias(es).
+
+  example: "Magecart Group 6"
```

```suggestion
  example: '[ "Magecart Group 6" ]'
  normalize:
    - array
```

Looks good! Let's also add context that this field is expected to be an array of values.